java-checks/src/test/java/org/sonar/java/filters/SuppressWarningFilterTest.java

```diff
@@ -69,4 +86,44 @@ void verify() {
     );
   }
 
+  @Test
+  void verify_2() {
+    FilterVerifier.verify("src/test/files/filters/SuppressWarningFilter_2.java", new SuppressWarningFilter(),
```

Comment:
Do you think it makes sense to move these files to java-checks-test-sources module?

Reply:
I tried the same way we do for unit tests, but it's not working. I'm not sure what is the exact issue, but at first glance, it doesn't seem trivial to fix.

It would be nice to have, but I don't think it's a must, I think we can let it as it is. Is it fine for you?

Reply:
Fine by me